### ISSUE TITLE
Is there a way to avoid reformatting at all?

### ISSUE THREAD

**Issue by da3x** [essiembre/eclipse-rbe#6]
I've just tried v1.0.2 with Eclipse Luna and found no obvious way to avoid reformatting or sorting at all. Do I miss something?

**Comment by fhoess** [comment]
Same problem here

**Comment by kraiz** [comment]
Forced formatting **creates** a real problem for merging 2 two branches with different max key length.

**Comment by k0ner** [comment]
This feature is scheduled for the next release.

**Comment by samirck** [comment]
Forced formatting is the only thing stopping me to use it.
Thank you for considering this issue.
:+1:

**Comment by anfe0690** [comment]
Yes please, I can't make the comments stay with their respective lines.

**Comment by aaron13100** [comment]
When is the next release scheduled?

**Comment by Roby73** [comment]
Hi all. Also for me it is a problem!
My solution should be to insert some sections block markers (ex. # [Section1-begin] ... #[Section1-end])  and use them to split the file into sorted sections. In the plugin, some options could allow to user to enable/disable the feature and toggle the view from splitted sections / all in a view.
Thanks and best regards.
R.A.

**Comment by rbi** [comment]
I just installed the plugin and edited property files in my project with it. It changed every single line of my property files because it sorted them alphabetically. I had to do a `git reset` and uninstall the plugin. It's a pity because the looked nice otherwise.

**Comment by sunil-samuel** [comment]
This still hasn't been implemented?  Everything else is great with this plugin, there should be a configuration parameter that you can check or uncheck to enable/disable sorting.  Unfortunately, I'm uninstalling this plugin for now.
